apps/feedback/test/mailer_test.exs

```diff
@@ -54,7 +54,10 @@ defmodule Feedback.MailerTest do
     end
 
     test "uses the comments of the message for the description" do
-      Mailer.send_heat_ticket(%{@base_message | comments: "major issue to report"}, nil)
+      Mailer.send_heat_ticket(
+        %{@base_message | comments: "major issue to report"},
+        nil
+      )
```

Comment:
Looks like this change is now formatting-only? Could revert it to minimize diff noise